extension/background.js

```diff
@@ -235,6 +250,65 @@ function getOperatingSystem() {
   });
 }
 
+// TRIGGERING THE FIRST SURVEY (HTTP OR HTTPS)
+
+/**
+ * Ensures the first (HTTP vs HTTPS) survey is completely ended.
+ */
+function endFirstSurvey() {
```

Comment:
Does end mean completed by the user? If so, could you add a comment along those lines?